diff --git a/dv_utils/dv_test_status_pkg.sv b/dv_utils/dv_test_status_pkg.sv
index d1516a2..3d7814e 100644
--- a/dv_test_status_pkg.sv
+++ b/dv_test_status_pkg.sv
@@ -10,13 +10,6 @@ package dv_test_status_pkg;
   // signature along with a banner. The signature can be used by external scripts to determine if
   // the test passed or failed.
   function automatic void dv_test_status(bit passed);
-`ifdef INC_ASSERT
-    if (prim_util_pkg::end_of_simulation) begin
-      // The first arg '1' is the error code, arbitrarily set to 1.
-      $fatal(1, "prim_util_pkg::end_of_simulation was already signaled!");
-    end
-    prim_util_pkg::end_of_simulation = 1'b1;
-`endif
     if (passed) begin
       $display("\nTEST PASSED CHECKS");
       $display(" _____         _                                  _ _ ");
